config/routes.rb

```diff
@@ -175,6 +175,7 @@
     resource :declaration, only: %i[show]
     resource :invalid_login, only: :show
     resource :invalid_schedules, only: :show
+    resource :user_not_founds, only: :show
```

Comment:
Do we definitely want this pluralised? Is that a silly question - is that the correct rails convention?